<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Система мониторинга</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Фильтрация экстремумов</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBox1">
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="maximum">
           <double>99.989999999999995</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>0.050000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>163</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Начальное значение</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Порог значения тренда, %</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="spinLevel">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_3">
          <property name="maximumSize">
           <size>
            <width>153</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Конечное значение</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_x1">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>9999.989999999999782</double>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_x2">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>9999.989999999999782</double>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QCheckBox" name="checkMax">
          <property name="text">
           <string>Максимум</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QCheckBox" name="checkMin">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Минимум</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>116</width>
          <height>76</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_4">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>40</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Derivative&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_3">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>40</height>
         </size>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Minimum&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser_2">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>40</height>
         </size>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Maximum&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCustomPlot" name="widget" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>586</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_3"/>
    <addaction name="action_16"/>
    <addaction name="action_12"/>
    <addaction name="action_8"/>
    <addaction name="action_11"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Экстремумы функции</string>
    </property>
    <addaction name="action_10"/>
    <addaction name="action_9"/>
    <addaction name="action_7"/>
    <addaction name="action_13"/>
    <addaction name="action_21"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Технический анализ</string>
    </property>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>Линия тренда</string>
     </property>
     <addaction name="action_5"/>
    </widget>
    <addaction name="menu_5"/>
    <addaction name="actionD"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/img/img/open.png</normaloff>:/img/img/open.png</iconset>
   </property>
   <property name="text">
    <string>Выбрать Файл</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Производная</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>График функции</string>
   </property>
  </action>
  <action name="action_4">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Тестовый график</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Фильтрация</string>
   </property>
   <property name="toolTip">
    <string>выбор фильтра или значения</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>Новый эксперимент</string>
   </property>
  </action>
  <action name="action_11">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/img/img/exit.png</normaloff>:/img/img/exit.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_12">
   <property name="icon">
    <iconset resource="source.qrc">
     <normaloff>:/img/img/512px-Save_font_awesome.svg.png</normaloff>:/img/img/512px-Save_font_awesome.svg.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить график</string>
   </property>
  </action>
  <action name="action_18">
   <property name="text">
    <string>Метод наименьших квадратов</string>
   </property>
  </action>
  <action name="action_20">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>Метод наименьших квадратов</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>Очистить графики</string>
   </property>
  </action>
  <action name="action_21">
   <property name="text">
    <string>График скорости изменения параметра</string>
   </property>
  </action>
  <action name="actionD">
   <property name="text">
    <string>Δ сигнала</string>
   </property>
  </action>
  <action name="action_24">
   <property name="text">
    <string>Начальное значение диапазона</string>
   </property>
  </action>
  <action name="action_26">
   <property name="text">
    <string>Конечное значение диапазона</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Удалить точки экстремумов</string>
   </property>
  </action>
  <action name="action_9">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Установить вручную</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Метод наименьших квадратов</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>Автопоиск</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>Скорость изменения параметра</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
